### ISSUE TITLE
Quote marks colored differently from text in triple-quoted Python strings

### ISSUE THREAD

**Issue by jordanweaver** [jamiewilson/predawn-syntax#11]
I haven't checked if this is relevant/present for other languages, but in Python, using triple-quoted strings with double quote marks (`"`) results in different coloring for the `"""` than that of the quoted text. It seems like it almost gets it right (compared to Sublime's version of the Predawn theme) in that the quoted text is rendered grey (as expected) but the marks stay blue. Having the fully grey distinction is helpful in docstrings.

In Atom:
![image](https://cloud.githubusercontent.com/assets/423534/13450539/2380e80a-dffc-11e5-94fa-a8bd48d89d11.png)

In Sublime:
![image](https://cloud.githubusercontent.com/assets/423534/13450519/f0897e30-dffb-11e5-9c8d-ed0ce0a0712c.png)

**Comment by jamiewilson** [comment]
Ah. hadn't seen that before. Thanks for filing. Just to be clear, the triple quoted line—and all quote marks—should be blue, right?

**Comment by jordanweaver** [comment]
The Sublime theme rendered them all grey like in the second screenshot. I think this helped distinguish generic triple quoted strings that typically use single-quotes (`'''`) from docstrings in a method/func, which traditionally use the double-quotes (`"""`). Though I suppose this is a matter of taste (it is your theme after all!), but if the goal is to be consistent with Sublime then I vote for all grey in that specific case. Thanks for putting this together BTW. It was nice to see something familiar when I moved from Sublime to Atom.

**Comment by jamiewilson** [comment]
Ah, okay, sorry, I thought that triple quotes were just a way to format complex strings, but if they are more like comments, then they should definitely be grey.

Glad you like it on Atom! Sorry for the inconsistencies, and let me know if you catch any others.

**Comment by jordanweaver** [comment]
> Ah, okay, sorry, I thought that triple quotes were just a way to format complex strings, but if they are more like comments

Yeah, that's the gist of it. They can be used for either purpose but you often seen them used for code documentation since they can span multiple lines without escape chars. And as far as Python is concerned, it might be worth formatting the `'''` case the same way since the language doesn't make any distinction between the two.

``` python
''' 
Example of a
multiline comment-style 
string
'''

def my_func():
    """ 
    Another example used as a docstring
    """
    return something

my_str = "standard string"
```

**Comment by nmat** [comment]
To get the "blue" one has to add a line like:

```
.syntax--string.syntax--quoted.syntax--double.syntax--single-quoted.syntax--block.syntax--python {
  color: #777777;
}
```

Which is located in if you edit the themes styles/base.less file
